main.js

```diff
@@ -8,4 +9,59 @@ document.querySelector('#menu').addEventListener('click', () => on());
 document.querySelector('.close-icon').addEventListener('click', () => off());
 document.querySelectorAll('.text-a').forEach((item) => {
   item.addEventListener('click', () => off());
-});
\ No newline at end of file
+});
+/* Dynamic Project Functions */
+function projectOn() {
+  document.getElementById('output').style.display = 'block';
+}
+function projectOff() {
+  document.getElementById('output').style.display = 'none';
+}
+function getProject(a) {
```

Comment:
- [ ] I would suggest you use meaningful variable names, in this case `"a"` doesn't give any hint about what it contains, please be a little more specific as this would improve your code readability tremendously! 🤓 🛠️ 

💡 _Hint: Since `"a"` in this case refers to the index of the project, a good name for this variable could be something like: `"projectIndex"`_